dynatrace_extension/cli/create/extension_template/setup.py.template

```diff
@@ -1,7 +1,20 @@
+from pathlib import Path
 from setuptools import setup, find_packages
 
+
+def find_version() -> str:
+    extension_yaml_path = Path(__file__).parent / "extension" / "extension.yaml"
+    version = "0.0.0"
+    with open(extension_yaml_path) as f:
```

Comment:
Have we tested that it works when deploying to EEC?

I don't think `extension.yaml` gets added as part of the python code, we need to make sure it is inside the WHL file or `pip install` will fail

Reply:
FYI @vduseev